modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/impl/DefaultLogStorageFactory.java

```diff
@@ -212,6 +214,17 @@ public LogStorage createLogStorage(String groupId, RaftOptions raftOptions) {
         return new RocksDbSharedLogStorage(this, db, confHandle, dataHandle, groupId, raftOptions, executorService);
     }
 
+    @Override
+    public boolean destroyLogStorage(String uri) {
+        try {
+            RocksDbSharedLogStorage.destroyAllEntriesBetween(db, confHandle, dataHandle, groupStartPrefix(uri), groupEndPrefix(uri));
+            return true;
+        } catch (RocksDBException e) {
+            LOG.error("Fail to destroy the log storage for {}.", e, uri);
```

Comment:
Why do you think it's not worth throwing an exception?

Reply:
This is how all methods in JRaft work: they don't throw, just log and return `false`

Reply:
omg, ok